Makefile

```diff
@@ -0,0 +1,48 @@
+# Run `make help` to display help
+
+# --- Global -------------------------------------------------------------------
+
+all: check format ## check and format
+	@if [ -e .git/rebase-merge ]; then git --no-pager log -1 --pretty='%h %s'; fi
+	@echo '$(COLOUR_GREEN)Success$(COLOUR_NORMAL)'
+
+check:  ## Run shellcheck
+	shellcheck *.sh
+
+format:  ## Format shell scripts. Auto-fix with shfmt -i 4 -w *.sh
```

Comment:
This target is not well named I think. I would expect it to actually format, but instead it just checks the format. I would add `shfmt -i 4 -d *.sh` to the `check` target above and make this one do `-w` so it is easy to format the scripts.

Reply:
great idea.